community/kernel/src/main/java/org/neo4j/kernel/api/DataRead.java

```diff
@@ -116,24 +112,25 @@ long nodeGetFromUniqueIndexSeek( IndexDescriptor index, Object value ) throws In
      */
     PrimitiveIntIterator nodeGetLabels( long nodeId ) throws EntityNotFoundException;
 
-    PrimitiveIntIterator nodeGetAllPropertiesKeys( long nodeId ) throws EntityNotFoundException;
+    PrimitiveIntIterator nodeGetPropertyKeys( long nodeId ) throws EntityNotFoundException;
+
+    PrimitiveIntIterator relationshipGetPropertyKeys( long relationshipId ) throws EntityNotFoundException;
 
-    PrimitiveIntIterator relationshipGetAllPropertiesKeys( long relationshipId ) throws EntityNotFoundException;
+    PrimitiveIntIterator graphGetPropertyKeys();
```

Comment:
What graphs we are talking about in `graphGetPropertyKeys`, `graphHasProperty` and `graphGetProperty`?